src/state/index.ts

```diff
@@ -24,7 +24,13 @@ const store = configureStore({
     multicall,
     lists
   },
-  middleware: [...getDefaultMiddleware({ thunk: false }), save({ states: PERSISTED_KEYS })],
+  middleware: [
+    ...getDefaultMiddleware({
+      thunk: false,
+      immutableCheck: false
```

Comment:
this helps a lot?

Reply:
It changes a lot. I assume this is showing that they might have some issues with deep objects in the middleware and checks are expensive

The performance is horrible because of this middleware, since I don't plan to fix their state issue (at this point), I'll prefer to disabled it for now. 
If we think is useful, we can always allow later for just our stores